file-native/src/main/java/org/ballerinalang/stdlib/file/service/FSListener.java

```diff
@@ -52,7 +50,7 @@
     private BObject service;
     private Map<String, MemberFunctionType> attachedFunctionRegistry;
     private static final StrandMetadata ON_MESSAGE_METADATA = new StrandMetadata(BALLERINA_BUILTIN_PKG_PREFIX,
```

Comment:
We can use the `ModuleUtils.getModule().getOrg()` for this, right?

Reply:
+1. Fixed